### ISSUE TITLE
How to implement other payment methods

### ISSUE THREAD

**Issue by studio1902** [edalzell/statamic-charge#45]
When using creditcards you need to post to /!/Charge/subscription. How do you implement other payment methods that have a different flow from credit cards. For example iDeal goes to an external page and then redirects back to your own site using a return_url. 

How and where do you handle payment results with Charge this way?

**Comment by edalzell** [comment]
Do you mean still using Stripe?

**Comment by studio1902** [comment]
Yeah, using Stripe and Elements. As the flow is different for other payment methods. I wouldn't know how to implement them using Charge.

**Comment by studio1902** [comment]
I just noticed that 'payment_method_types' is currently hardcoded on the PHP side. Does this mean other payment types aren't supported or is this just WIP.

**Comment by edalzell** [comment]
Card only at the moment, sorry. Step one, get it working. THEN expand, if needed.

**Comment by studio1902** [comment]
Right, good to know. Will not try to implement then.

**Comment by edalzell** [comment]
@studio1902 next version sits on top of Larvel's Cashier, is it supported in Cashier?
